### ISSUE TITLE
Cthun Eye Beam Changing Targets on second Beam instead of Third

### ISSUE THREAD

**Issue by ZachRobins** [TeamEverlook/Everlook-BugTracker#342]
**Describe the bug**
Cthuns eye beam is targeting a new player after a single beam instead of two beams at the start of the fight. 

**Quest/Item/Spell Link**
If this is quest, item or spell related, please put the link here.

https://www.wowhead.com/classic/spell=26134/eye-beam

Example:  https://www.wowhead.com/classic/quest=5904/a-plague-upon-thee

**To Reproduce**

Aggro Cthun with a few eligible targets. Cthun should target his initial target for one eye beam and then select another player instead of the first player.

**Expected behavior**
A clear and concise description of what you expected to happen
The first two eye beams should target the same player before selecting another player for the third beam.

**Screenshots**
have video in discord ticket, not able to upload anywhere at the moment

here is a Warcraft log showing it not targeting Disq(the player to pull) on the second beam three times.

https://vanilla.warcraftlogs.com/reports/pZgH7kGL8TcJPCFA#fight=last&type=damage-taken&ability=26134&view=timeline

**Additional context**

**Comment by Solation1** [comment]
Yeah this issue is rather frustrating, happened a couple of vmangos patches ago
